ops/demo-deploy.tmpl.yaml

```
@@ -35,6 +37,9 @@ extraVolumeMounts: &volMounts
   - name: uploads
     mountPath: /app/samvera/hyrax-webapp/storage/files
     subPath: storage-files
+  - name: logs
```

```suggestion
  - name: uploads
```
I suggest you just use the uploads mount and not try to make a new mount